charts/redpanda/values.schema.json

```diff
@@ -635,22 +635,20 @@
           "type": "array"
         },
         "topologySpreadConstraints": {
-          "type": "object",
-          "required": [
-            "maxSkew",
-            "topologyKey",
-            "whenUnsatisfiable"
-          ],
-          "properties": {
-            "maxSkew": {
-              "type": "integer"
-            },
-            "topologyKey": {
-              "type": "string"
-            },
-            "whenUnsatisfiable": {
-              "type": "string",
-              "pattern": "^(ScheduleAnyway|DoNotSchedule)$"
+          "type": "array",
+          "minItems": 1,
```

Comment:
Would there be any objections to setting that to 0?

Reply:
I think `0` is fine.